lib/login_gov/hostdata/s3.rb

```diff
@@ -0,0 +1,39 @@
+require 'aws-sdk-core'
+require 'fileutils'
+require 'json'
+
+module LoginGov
+  module Hostdata
+    class S3
+      attr_reader :bucket, :env
+
+      def initialize(bucket:, env:, s3_client: nil)
+        @bucket = bucket
+        @env = env
+        @s3_client = s3_client
+      end
+
+      def download_configs(configs)
+        configs.each do |s3_path, local_path|
+          download_config(s3_path, local_path)
+        end
+      end
+
+      private
+
+      def s3_client
+        @s3_client ||= Aws::S3::Client.new
```

Comment:
Do we need region here? Or do instances automatically pick the region that they're currently running in?

Reply:
My understanding is the AWS SDK automatically picks the region it's running in via env vars and magic stuff? But it's easy to make that a param if you're not sure?

Reply:
Pulled in region in 4c6af40ec72917a81d7f6efa5ecafd18d085e55f